accounts/manager.go

```diff
@@ -93,6 +106,16 @@ func (am *Manager) Config() *Config {
 	return am.config
 }
 
+// AddBackends starts the tracking of additional backends for wallet updates.
+// cmd/geth assumes once this func returns the backends have been already integrated.
+func (am *Manager) AddBackends(backends ...Backend) {
```

Comment:
I think it would be cleaner to have just `AddBackend`. The constructor needs all the backends in one batch, but if we support adding them from the outside async, might as well add one-by-one.